src/main/java/app/entities/unitconverter/AreaConverter.java

```diff
@@ -0,0 +1,70 @@
+package app.entities.unitconverter;
+
+
+public class AreaConverter
+{
+    public double squareMetersToSquareKilometers(double squareMeters)
+        {
```

Comment:
this indentation is something that is usually addressed in these PRs. 

We did however not settle on a code convention, but when we do this will not be it 😉